src/server/req_quejob.c

```diff
@@ -884,12 +883,6 @@ req_quejob(struct batch_request *preq)
 						free(pj->ji_wattr[(int)JOB_ATR_outpath].at_val.at_str);
 						pj->ji_wattr[(int)JOB_ATR_outpath].at_val.at_str = result;
 					}
-				} else if (pj->ji_wattr[(int)JOB_ATR_outpath].at_val.at_str[l -1] == '/') {
```

Comment:
I agree on standardizing the file names. But at the same time I feel formation of file names deserves to be in the server side than in host.
The prepared path just before the job execution is visible to the hook and other runtime monitoring tools which will no longer work if we move this logic to MOM.

Reply:
It will be difficult to incorporating this comment while supporting arrayjobs, The hook can anyway form the same name with the logic used here. If you still dont agree, please suggest how to implement this bug fix